bindings/experimental/distrdf/python/DistRDF/Backends/Dask/Backend.py

```diff
@@ -27,9 +27,13 @@
 class DaskBackend(Base.BaseBackend):
     """Dask backend for distributed RDataFrame."""
 
-    def __init__(self, daskclient=None):
+    def __init__(self, daskclient):
         super(DaskBackend, self).__init__()
-        self.client = daskclient
+        # If the user didn't explicitly pass a Client instance, the argument
+        # `daskclient` will be `None`. In this case, we create a default
+        # dask client to run the computations.
```

Comment:
`we create a default Dask client for a cluster of N worker processes, where N is the number of cores on the local machine`.